tests/eas/acceptance.py

```diff
@@ -54,15 +54,20 @@ def setUpClass(cls, *args, **kwargs):
     def _experimentsInit(cls, *args, **kwargs):
         super(EasTest, cls)._experimentsInit(*args, **kwargs)
 
-        if SET_IS_BIG_LITTLE:
+        # Try writing a value but do not report an error if it fails; these
+        # flags do not exist on all kernels.
+        def write_maybe(path, value):
             try:
-                cls.target.write_value(
-                    "/proc/sys/kernel/sched_is_big_little", 1)
-            except TargetError:
-                # That flag doesn't exist on mainline-integration kernels, so
-                # don't worry if the file isn't present.
+                cls.target.write_value(path, value)
+            except TargetError as e:
                 pass
 
+        if SET_INITIAL_TASK_UTIL:
+            write_maybe("/proc/sys/kernel/sched_initial_task_util", 1024)
```

Comment:
Given the previous comment we can probably avoid to add this write_maybe function...